### PR TITLE
support fomat with blocksize and chunksize

### PR DESCRIPTION
**Pull request by wu-hanqing**

### CODE REVIEW

File: internal/task/scripts/shell/format.sh [hunk 1]
```diff
@@ -9,11 +9,14 @@ percent=$2
...
+chunkfile_block_size=$6
 
 mkdir -p $chunkfile_pool_dir
 $binary \
   -allocatePercent=$percent \
   -fileSize=$chunkfile_size \
   -filePoolDir=$chunkfile_pool_dir \
   -filePoolMetaPath=$chunkfile_pool_meta_path \
-  -fileSystemPath=$chunkfile_pool_dir
+  -fileSystemPath=$chunkfile_pool_dir \
```
Comment (Wine93) [inline internal/task/scripts/shell/format.sh:20]: Will the command report invalid option if user use the old container image which not support specify the `fileSystemPath`?

File: internal/task/scripts/shell/format.sh [hunk 2]
```diff
@@ -9,11 +9,14 @@ percent=$2
...
+  -blockSize=$chunkfile_block_size \
```
Comment (Wine93) [inline internal/task/scripts/shell/format.sh:21]: ditto.

File: internal/task/scripts/shell/format.sh (hunk 1 above)
Comment (wu-hanqing) [reply internal/task/scripts/shell/format.sh:20]: below `-undefok` is a workaround for this, older format tools won't report error.

File: internal/task/task/bs/format.go [hunk 3]
```diff
@@ -188,8 +190,8 @@ func NewFormatChunkfilePoolTask(curveadm *cli.CurveAdm, fc *configure.FormatConf
 	chunkfilePoolRootDir := layout.ChunkfilePoolRootDir
 	formatScript := scripts.FORMAT
 	formatScriptPath := fmt.Sprintf("%s/format.sh", layout.ToolsBinDir)
-	formatCommand := fmt.Sprintf("%s %s %d %d %s %s", formatScriptPath, layout.FormatBinaryPath,
-		usagePercent, DEFAULT_CHUNKFILE_SIZE, layout.ChunkfilePoolDir, layout.ChunkfilePoolMetaPath)
+	formatCommand := fmt.Sprintf("%s %s %d %d %s %s %d", formatScriptPath, layout.FormatBinaryPath,
+		usagePercent, chunkSize, layout.ChunkfilePoolDir, layout.ChunkfilePoolMetaPath, blockSize)
```
Comment (ilixiaocui) [inline internal/task/task/bs/format.go:194]: Now block size exist in chunkserver.conf, mds.conf, curveadm format, hoow to ensure the unity of them?

Comment (wu-hanqing) [reply internal/task/task/bs/format.go:194]: > Now block size exist in chunkserver.conf, mds.conf, curveadm format, hoow to ensure the unity of them?

If chunkfilepool is enabled, blocksize in chunkserver.conf will be overwritten from chunkfilepool's meta which is created by curveadm.

And when deploying the cluster, in the registering process chunkserver will report its blocksize to mds, if blocksize is different between chunkserver and mds, the register will fail.

**Review by ilixiaocui — APPROVED** [review]

**Pull request merged**